### PR TITLE
Add FAX.PLUS

### PR DESCRIPTION
**Pull request by alohi**

### CODE REVIEW

**Comment by alohi** [comment]
When should we expect to have one of your reviewers approve our request?

**Comment by kmpoppe** [comment]
Dear @alohi, this project is maintained purely by volunteers, thus the time spared by @2factorauth/collaborators is limited. I therefore ask for your patience.
Thanks for your understanding!
// Kai

**Review by kmpoppe — DISMISSED** [review]

**Review by kmpoppe — APPROVED** [review]

File: _data/other.yml [hunk 1]
```diff
@@ -212,6 +212,13 @@ websites:
       - software
     doc: https://support.everlaw.com/hc/en-us/articles/206312165
 
+  - name: FAX.PLUS
+    url: https://www.fax.plus
+    img: faxplus.png
+    tfa:
+      - software
```
Comment (Carlgo11) [inline _data/other.yml]:
```suggestion
      - totp
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/other.yml (hunk 1 above)
Comment (kmpoppe) [reply _data/other.yml]: Resolved with 94e0792. Why didn't I see that 😐

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**